CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to Libero
+We encourage contributions of any kind.  By participating in this project, you agree to abide by the [code of conduct](https://libero.pub/code-of-conduct/).
+
+Contributions of code or documentation are covered by our [Contributor License Agreement (CLA)](https://libero.pub/governance/cla/). Please [sign the CLA](https://libero.pub/sign-cla) or ensure your employer has signed the CLA on your behalf.
+
+All Libero repositories will have further instructions in their individual `CONTRIBUTING.md` with more specific details.
```

Comment:
'All Libero repositories will have further details in their own `CONTRIBUTING.md` file.'?